src/plugins/janus_videoroom.c

```diff
@@ -7103,6 +7103,9 @@ static void janus_videoroom_recorder_create(janus_videoroom_publisher_stream *ps
 					janus_videoroom_media_str(ps->type));
 			}
 		}
+		/* If the stream has a description, store it in the recording */
+		if(ps->description && rc)
+			rc->description = g_strdup(ps->description);
```

Comment:
I'd prefer there to be a setter in `record.h` for the description, just as in other properties. I don't like the idea of plugins messing with the properties of the `janus_recorder` directly, as a getter can use a mutex for instance.

Reply:
Fixed